### DISCUSSION TITLE
vulnerabilityreports is only in kube-system namespace

### DISCUSSION THREAD

**Discussion by gtkemal** [aquasecurity/trivy#7919] [Q&A]
### Question

i installed trivy-operator with helm  but  vulnerabilityreports is in only in kube-system namespace

### Target

None

### Scanner

None

### Output Format

None

### Mode

None

### Version

```bash
v0.22
```

**Comment by knqyf263** [comment]
You can open a report here.
https://github.com/aquasecurity/trivy-operator
